atlas/config_local.py.example

```diff
@@ -10,6 +10,8 @@ LOG_LOCATION = '/data/logs/atlas.log'
 
 # List of usernames that are allowed to interact with the API
 ALLOWED_USERS = ['sample_user', 'other_user']
+TEST_ACCOUNTS = ['', '', '']
```

Comment:
This will also require a corresponding PR in our Ansible repo to update the config file for our environments